### PR TITLE
Simplify ItemsHandler and ActivityHandler API

### LINKED ISSUE #3445: Types don't match if using `csv` as return type
When returning `csv` types are not correct

```ts
sdk.items("posts").read({ fields: ["*"], export: "csv" })
.then((v) => console.log(v));
```
Prints `csv` string, but types for `v` are `v: Response<Record<string, any> | Record<string, any>[]>`

### PR DESCRIPTION
**Pull request by apstanisic**
closes #3445

### CODE REVIEW

File: packages/sdk-js/tsconfig.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
 	"compilerOptions": {
 		"target": "es5",
-		"lib": ["es5", "DOM"],
```
Comment (rijkvanzanten) [inline packages/sdk-js/tsconfig.json]: Didn't you mention _adding_ DOM in that other PR? 🤔

Comment (apstanisic) [reply packages/sdk-js/tsconfig.json]: Yes,  #3479. That's why I removed dom, I was hoping that @nickrum will add it. I'll add another commit with added dom

Comment (apstanisic) [reply packages/sdk-js/tsconfig.json]: I added dom in last commit

Comment (nickrum) [reply packages/sdk-js/tsconfig.json]: Sorry, I was a little busy today.

**Pull request closed without merging**